CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 # Unreleased
 
+- **Breaking:** No longer export `platform::x11::XNotSupported`.
```

Comment:
@notgull I couldn't see a reason why this was exported, so I removed it, but maybe you can shed some light on it?